### DISCUSSION TITLE
V0.7.10(Web1.5.7.1)

### DISCUSSION THREAD

**Discussion by cloud-fs** [cloud-fs/cloud-fs.github.io#127] [Announcements]
1. Web界面增加文件复制功能：点击复制按钮，可以在同盘内或者跨盘快速复制文件。跨盘复制的功能和原来的移动/复制到...按钮功能完全一致，后面版本会把移动和复制完全分开；同网盘内复制功能优先调用网盘API实现快速复制，目前123网盘API不支持复制，所以123网盘内复制和跨盘复制功能一样，由CloudDrive2遍历复制的文件夹采用秒传方式实现；因为无法确定WebDAV Server是否支持复制接口，所以添加的WebDAV网盘也采用由CloudDrive2遍历复制的文件夹上传文件的方式实现；因API限制，Google Drive的同盘内复制只支持文件复制，不支持文件夹复制
2. WebDAV Server支持文件复制接口
3. 增强的文件系统变更通知功能：网盘目录缓存时间到期后通过cd2以任何方式访问网盘目录，或者Web界面点击刷新按钮时，如果发现网盘被第三方修改的文件变更，会发出文件变更通知。使用该功能建议打开目录缓存本地持久化，否则目录缓存到期后会被清除而无法检测到文件更新
4. 115账号改用UID作为用户唯一标识，修复相同网名的不同115账号不能同时添加的问题，已添加的115账号会自动更新，不需要重新登录
5. 提高目录缓存本地持久化效率，修复本地持久化的若干bug
6. 修复了0.7.10-beta3的所有已知bug，如果发现目录缓存本地持久化出现各种问题，请先关闭目录缓存本地持久化开关然后再打开，这将清空数据库重新建立缓存库
7. 其他bug修复和性能优化

<hr /><em>This discussion was created from the release <a href='https://github.com/cloud-fs/cloud-fs.github.io/releases/tag/v0.7.10'>V0.7.10(Web1.5.7.1)</a>.</em>

**Comment by andymachs** [comment]
客户端和服务器端应该要支持自动更新！
